source/functions/initializeWecSim.m

```diff
@@ -94,13 +94,17 @@
         end
     end; clear n combine;
     try 
-        waves.spectrumFile = ['..' filesep pctDir filesep '..' filesep waves.spectrumFile];
-        waves.elevationFile = ['..' filesep pctDir filesep '..' filesep waves.elevationFile];
+        for iW = 1:length(waves)
+            waves(iW).spectrumFile = ['..' filesep pctDir filesep '..' filesep waves(iW).spectrumFile];
+            waves(iW).elevationFile = ['..' filesep pctDir filesep '..' filesep waves(iW).elevationFile];
+        end
     end
 end
 
 % Waves and Simu: check inputs
-waves.checkInputs();
+for iW = 1:length(waves)
```

Comment:
@salhus Minor comment but can you update the `stopWECSim.m` to remove the iW variable that you generate through the for loop.

Reply:
Thanks @nathanmtom 
- done